lib/events/base_event.dart

```diff
@@ -38,7 +37,7 @@ class BaseEvent extends EventOptions {
     String? appSetId,
     String? androidId,
     String? language,
-    String library = "${Constants.packageName}/${Constants.packageVersion}",
```

Comment:
Can we remove `Constants.packageVersion` to prevent confusion now that it switched to BuildConfig?

Another option would be to have semantic release update the packageVersion constants.dart instead of the build.gradle.

Reply:
I updated constants at https://github.com/amplitude/Amplitude-Flutter/pull/175/commits/5d599f746d8e598c322c633b970f4f442ccd33b5.

I'd like to keep semantic release to update build.gradle as I don't find a way for Android plugin to read from constants.dart.